docs/migrating_to_6.md

```diff
@@ -330,6 +331,33 @@ If you set `timestamps: true`, Mongoose will now make the `createdAt` property `
 
 `safe` is no longer an option for schemas, queries, or `save()`. Use `writeConcern` instead.
 
+<h3 id="schematype-set-parameters"><a href="#schematype-set-parameters">SchemaType `set` parameters</a></h3>
+
+SchemaType `set` parameters now use `priorValue` as the second parameter instead of `self`.
```

Comment:
Sounds good, what do you think instead of using `self` we use `schemaType` instead?

My only concern would be that people would not know what `schemaType` actually is, it's sort of a mongoose internal, but even then I think `schemaType` is more descriptive.